test/test_sfcc_cim_object_path.rb

```diff
@@ -8,8 +8,14 @@
     @op = Sfcc::Cim::ObjectPath.new('root/cimv2')
   end
 
-  it 'should have a nil class name' do
-    assert_nil @op.classname
+  it 'should raise on a nil class name' do
+    assert_raises ArgumentError do
+      Sfcc::Cim::ObjectPath.new("root/cimv2", nil)
+    end
+  end
+
+  it 'should have an empty class name' do
+    assert_equal "", @op.classname
   end
 end
 
```

Comment:
I am trying from now to follow the ruby style by running rubocop on the files. Here single quotes should be used. I fix it in another commit, but in the future lets try to run rubocop on the files before.